spellbook/macros/optimize_tables.sql

```diff
@@ -0,0 +1,32 @@
+{% macro optimize_tables() %}
+{%- if flags.full_refresh -%}
+{% set sql_1 %}
+OPTIMIZE dbt_thomas_transfers_ethereum.erc20_agg_hour;
+{% endset %}
+
+{% set sql_2 %}
+OPTIMIZE transfers_ethereum.erc20_agg_day;
+{% endset %}
+
+{% set sql_3 %}
+OPTIMIZE opensea_ethereum.trades;
+{% endset %}
+
+{% set sql_4 %}
+OPTIMIZE opensea_solana.trades;
+{% endset %}
+
+{% set sql_5 %}
+OPTIMIZE magiceden_solana.trades;
+{% endset %}
+
+{% do run_query(sql_1) %}
```

Comment:
slight preference for the set sql to be named after the optimize. 
so for instance `sql_4` -> `optimize_opensea_solana`

Reply:
Agreed, done !